packages/sdk-core/src/modules/database/BacktraceDatabase.ts

```diff
@@ -135,16 +135,14 @@ export class BacktraceDatabase extends Events<BacktraceDatabaseEvents> implement
             return undefined;
         }
 
-        const sessionId = backtraceData.attributes?.['application.session'];
-
         const record: ReportBacktraceDatabaseRecord = {
             type: 'report',
             data: backtraceData,
             timestamp: TimeHelper.now(),
             id: IdGenerator.uuid(),
             locked: false,
             attachments: attachments,
-            sessionId: typeof sessionId === 'string' ? sessionId : undefined,
+            sessionId: this._sessionFiles?.sessionId,
```

Comment:
idea: you know when the file was created. Why not use a file creation date and drop the deprecated model at all?